src/components/PhoneNumberInput/Lazy.js

```diff
@@ -0,0 +1,19 @@
+import { h, Component } from 'preact'
+import style from './style.css'
+
+const Loading = () => <div className={style.loading}>Loading...</div>
+
+class PhoneNumberInputLazy extends Component {
```

Comment:
Can you explain what this is doing? I'm not sure I get the point of it

Reply:
This is where we perform the code splitting. I will add a comment for future reference.